WordPress/src/main/java/org/wordpress/android/ui/dataview/compose/RemoteImage.kt

```diff
@@ -20,6 +24,23 @@ fun RemoteImage(
             contentDescription = null,
             modifier = modifier
         )
+    } else if (imageUrl.startsWith("drawable:")) {
+        // Handle drawable resource ID passed as string
+        val resourceId = imageUrl.removePrefix("drawable:").toIntOrNull()
+        if (resourceId != null && isValidDrawableId(LocalContext.current, resourceId)) {
+            Image(
+                painter = painterResource(id = resourceId),
+                contentDescription = null,
+                modifier = modifier
+            )
+        } else {
+            // Fallback if resource ID can't be parsed or is invalid
+            Image(
+                painter = painterResource(id = fallbackImageRes),
+                contentDescription = null,
+                modifier = modifier
+            )
+        }
```

Comment:
⛏️ What about adding the if inside the painter assignment to avoid code duplication?

```suggestion
            Image(
                painter = painterResource(
                    id = if (resourceId != null && isValidDrawableId(LocalContext.current, resourceId)) {
                        resourceId
                    } else {
                        fallbackImageRes
                    }
                ),
                contentDescription = null,
                modifier = modifier
            )
```